app/models/host.rb

```diff
@@ -9,6 +9,7 @@ class Host < ApplicationRecord
   belongs_to :canonical_host, class_name: "Host"
 
   validates :hostname, presence: true
+  validates :hostname, uniqueness: { message: ->(_object, data) { "The hostname #{data[:value]} already exists" } }
```

Comment:
Thanks for sorting this out. Is there any way we could let the user know what they need to do if a hostname already exists? "No" is a perfectly fine answer, just want to make sure we're making it as usable as possible

Reply:
I think the only way to delete a host, is to delete a site (the stuff Cristina did). 

So this would either be: 

1. `The hostname www.cabinetoffice.gov.uk already exists. You must delete the cabinetoffice site to remove it` or
2. `The hostname www.cabinetoffice.gov.uk already exists. You must delete the associated site to remove it`

The problem with number 1 is that we use the default host throughout Transition to refer to a site, so it might be confusing. See pic below, the site is `cabinetoffice`, yet the heading is the default host `www.cabinetoffice.gov.uk`

<img width="820" alt="image" src="https://github.com/alphagov/transition/assets/47089130/162ff0de-65a7-43aa-ad04-9112473e51ec">

What do you think?

Reply:
@JonathanHallam Any thoughts on this? 🤔

Reply:
Sorry, should have got back to you sooner. I think number 2. I guess bonus points if it's possible to pull a link to the associated site in?

Reply:
We chatted about this in standup and went for option 1 so that the user can search for the existing site.

This is updated now if you can re-review please 🙏